trogon/widgets/parameter_controls.py

```diff
@@ -209,16 +211,12 @@ def make_widget_group(self) -> Iterable[Widget]:
         )
 
         # Get the types of the parameter. We can map these types on to widgets that will be rendered.
-        parameter_types = (
-            parameter_type.types
-            if isinstance(parameter_type, click.Tuple)
-            else [parameter_type]
-        )
+        parameter_types = [parameter_type] * schema.nargs if schema.nargs > 1 else [parameter_type]
```

Comment:
Use Python type over click-specific type.

Instead of relying on click-specific tuple types, let's determine the quantity of input boxes to create by referring to `schema.nargs`.